src/test/regress/expected/columnar_paths.out

```diff
@@ -569,5 +573,53 @@ BEGIN;
 (1 row)
 
 ROLLBACK;
+SET columnar.enable_qual_pushdown TO DEFAULT;
+BEGIN;
+SET LOCAL columnar.stripe_row_limit = 2000;
+SET LOCAL columnar.chunk_group_row_limit = 1000;
+CREATE TABLE correlated(x int) using columnar;
+INSERT INTO correlated
+  SELECT g FROM generate_series(1,100000) g;
+CREATE TABLE uncorrelated(x int) using columnar;
+INSERT INTO uncorrelated
+  SELECT (g * 19) % 100000 FROM generate_series(1,100000) g;
+COMMIT;
+CREATE INDEX correlated_idx ON correlated(x);
+CREATE INDEX uncorrelated_idx ON uncorrelated(x);
+ANALYZE correlated, uncorrelated;
+-- should choose chunk group filtering; selective and correlated
+EXPLAIN (analyze on, costs off, timing off, summary off)
+SELECT * FROM correlated WHERE x = 78910;
+                            QUERY PLAN
+---------------------------------------------------------------------
+ Custom Scan (ColumnarScan) on correlated (actual rows=1 loops=1)
+   Filter: (x = 78910)
+   Rows Removed by Filter: 999
+   Columnar Chunk Group Filters: (x = 78910)
+   Columnar Chunk Groups Removed by Filter: 99
+(5 rows)
+
+SELECT * FROM correlated WHERE x = 78910;
+   x
+---------------------------------------------------------------------
+ 78910
+(1 row)
+
+-- should choose index scan; selective but uncorrelated
+EXPLAIN (analyze on, costs off, timing off, summary off)
+SELECT * FROM uncorrelated WHERE x = 78910;
```

Comment:
Here, I think we can just do:

```sql
SELECT columnar_test_helpers.uses_index_scan (
$$
SELECT * FROM uncorrelated WHERE x = 78910;
$$
);
```

Reply:
Why do we favor that style in tests?

Reply:
actually it wasn't about the style, but was about using the helper function that we are mostly using in this file.

Reply:
In code, it's good to be very regular and organized; but for tests, too much organization can mean fewer code paths tested. The test seems to not be causing headaches so I'll embrace the variety. If it does cause some non-determinism or other headache, or if it's not testing the code path well, then we can change it.